.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,28 @@
+name: pytest
+
+on:
+  push:
+    branches: [main, develop, 73-initialize-pytest]
+  pull_request:
+    branches: [main, develop, 73-initialize-pytest]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Test with pytest
+        run: |
+          pytest
```

Comment:
This will install the top-level python dependencies, which are separate from our poetry venv and Django app.

You can look at the registration action that runs `poetry pytest` in the `/server/` folder:

https://github.com/bcgov/cas-registration/blob/16db8cf02c7cf429c2da23fe7772c227d595b05b/.github/workflows/test.yaml#L225